### PR TITLE
Update VSCode Telemetry Package to v0.6.2

### PR DESCRIPTION
**Pull request by WardenGnaw**

### CODE REVIEW

**Review by sean-mcmanus — APPROVED** [review]

File: Extension/package.json [hunk 1]
```diff
@@ -4561,7 +4561,7 @@
     "plist": "^3.0.5",
     "tmp": "^0.2.1",
     "vscode-cpptools": "^6.1.0",
-    "vscode-extension-telemetry": "^0.4.4",
+    "vscode-extension-telemetry": "^0.6.2",
```
Comment (bobbrow) [inline Extension/package.json]: I thought it changed to `@vscode/extension-telemetry` for 0.6.2

**Comment by sean-mcmanus** [comment]
It says "error Couldn't find any versions for "vscode-extension-telemetry" that matches "^0.6.2""

File: Extension/package.json (hunk 1 above)
Comment (WardenGnaw) [reply Extension/package.json]: It did. I forgot to change the package name :D

**Review by bobbrow — APPROVED** [review]

**Review by sean-mcmanus — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
@WardenGnaw I think yarn.lock needs to be updated too. And NOTICES.txt maybe as well.

**Pull request merged**